packages/stryker/src/MutatorFacade.ts

```diff
@@ -12,7 +12,15 @@ export default class MutatorFacade implements Mutator {
 
   mutate(inputFiles: File[]): Mutant[] {
     return MutatorFactory.instance()
-      .create(this.config.mutator, this.config)
+      .create(this.getMutatorName(this.config.mutator), this.config)
       .mutate(inputFiles);
   }
+
+  private getMutatorName(mutator: string | MutatorDescriptor) {
+    if (typeof mutator === 'string') {
+      return mutator;
+    } else {
+      return (mutator as MutatorDescriptor).name;
```

Comment:
This cast is not needed and can be removed. TypeScript's awesome type inference has your back.